pkg/hubble/dropeventemitter/dropeventemitter.go

```diff
@@ -74,16 +77,29 @@ func (e *DropEventEmitter) ProcessFlow(ctx context.Context, flow *flowpb.Flow) e
 		message := "Outgoing packet dropped (" + reason + ") to " +
 			e.endpointToString(flow.IP.Destination, flow.Destination) + " " +
 			e.l4protocolToString(flow.L4)
-		e.recorder.Event(&slimv1.Pod{
+
+		objMeta := metaslimv1.ObjectMeta{
+			Name:      flow.Source.PodName,
+			Namespace: flow.Source.Namespace,
+		}
+		if e.K8sWatcher != nil {
+			pod, err := e.K8sWatcher.GetCachedPod(flow.Source.Namespace, flow.Source.PodName)
+			if err == nil {
+				objMeta = metaslimv1.ObjectMeta{
+					Name:      flow.Source.PodName,
```

Comment:
why updating `Name` and `Namespace` required as you are updating in line 81

Reply:
Good catch. Simplifying this.